mypy 1.6.1 CI failure

`src/subscript/ri_wellmod/ri_wellmod.py:246: error: Incompatible types in assignment (expression has type "Instance | None", variable has type "bool")  [assignment]`

--- a/src/subscript/ri_wellmod/ri_wellmod.py
+++ b/src/subscript/ri_wellmod/ri_wellmod.py
@@ -241,7 +241,6 @@ def launch_resinsight(console_mode: bool, command_line_parameters: List[str]):
         wrapper = True
 
     # First launch attempt (always try twice, occasionally times out on busy nodes)
-    resinsight = False
     try:
         resinsight = rips.Instance.launch(
             resinsight_executable=resinsight_exe,
@@ -264,6 +263,7 @@ def launch_resinsight(console_mode: bool, command_line_parameters: List[str]):
             logger.critical(
                 "Failed to launch ResInsight (%s) again - stopping now.", resinsight_exe
             )
+            resinsight = None
 
     if wrapper:
         riexe_path.unlink()  # Delete wrapper

